### PR TITLE
Improved the user interface of the mod

### PR DESCRIPTION
**Pull request by TheGemDev**
Added uikit animations to menu 
Reduced text size
Screenshot (67)
Screenshot (66)

### CODE REVIEW

**Comment by TheGemDev** [comment]
@arthuro555 fixed the conflicts

File: Loader/webLoader/html/menu.html [hunk 1]
```diff
@@ -7,42 +7,46 @@
...
-        <h1 class="uk-heading-medium" id="connecting">Connecting to game...</h1>
-		<ul class="uk-subnav uk-subnav-pill" uk-switcher>
+        <h2 class="uk-heading-smalluk-card uk-card-default uk-card-body uk-animation-slide-top-small" id="connecting">Connecting to game...</h2>
+		<ul class="uk-subnav uk-subnav-pill uk-card uk-card-default uk-card-body uk-animation-slide-top-small " uk-switcher>
 			<li><a href="#">Overview</a></li>
 			<li id="modsList"><a href="#">Mod Manager</a></li>
 			<li id="scenesList"><a href="#">Scene switcher</a></li>
 			<li><a href="#">Settings</a></li>
 		</ul>
-		<ul class="uk-switcher uk-margin" uk-switcher="animation: uk-animation-fade">
```
Comment (arthuro555) [inline Loader/webLoader/html/menu.html:17]: You seem to have reverted some changes of commits on master. Is this intentional?

Comment (TheGemDev) [reply Loader/webLoader/html/menu.html:17]: not intentional. i would take a look

**Comment by arthuro555** [comment]
It seems there are conflicts again. Also, if the reversion of changes of master are mandatory, then I'll have to deny this PR as I want to keep those changes.

**Comment by TheGemDev** [comment]
> It seems there are conflicts again. Also, if the reversion of changes of master are mandatory, then I'll have to deny this PR as I want to keep those changes.

It's an improvement to the UI but if you want to keep the existing UI it's okay then I would close this PR 😄

**Pull request closed without merging**